packages/config/src/layer2s/common/assessCount.ts

```diff
@@ -0,0 +1,6 @@
+import { AssessCount } from '../types'
+
+export const subtractOne: AssessCount = (count) => count - 1
+
+export const subtractOneAfterBlock: (blockNumber: number) => AssessCount = () =>
```

Comment:
can you explain me this one? 😅

Reply:
This is a helper function, returning an AssessCount function. I'll rewrite this to be a bit more readable.